config/crd/bases/awx.ansible.com_awxs.yaml

```diff
@@ -1567,10 +1567,18 @@ spec:
                 description: Number of web instance replicas
                 type: integer
                 format: int32
+              web_unmanage_replicas:
```

Comment:
:fearful: maybe `task_manage_replicas` with default `true` bit less topsy turvy?
(e.g. all "affirmative" settings, to avoid managing double negatives)

Reply:
@TheRealHaoLiu , @rooftopcellist - Thoughts on this?  I think it may be a better approach?

Reply:
@kdelee - I have this switch to your suggestion.  I appreciate it!